packages/ember-data/lib/system/adapter.js

```diff
@@ -330,6 +330,14 @@ var Adapter = Ember.Object.extend({
 
     Serializes the record update and sends it to the server.
 
+    The updateRecord method is expected to return a promise that will
+    resolve with the serialized record. This allows the backend to
```

Comment:
hey, it's actually worked for me with **pre** serialized data instead of serialized data (the raw object before normalizing the model by serializer)? am i did something wrong or is it mistake in the docs?